content/altinn-studio/what-do-you-get/_index.nb.md

```diff
@@ -74,16 +74,12 @@ ha tilgang, og hvilke handlinger som skal være tilgjengelig for hvem. Foreløpi
 for:
 - Datautfylling
 - Signering
-- Betaling*
+- Betaling
 - Tilbakemelding
 - Oppsummering
 
 PDF for kvittering genereres automatisk.
 
-{{% notice info %}}
-\* Støtte for [brukerbetaling for tjenester](https://github.com/digdir/roadmap/issues/80) er under arbeid.
```

Comment:
\* Vi jobber med støtte for [brukerbetaling for tjenester](https://github.com/digdir/roadmap/issues/80).